### ISSUE TITLE
Suggest a dataset form: capture dataholder's country

### ISSUE THREAD

**Issue by kbraak** [gbif/portal16#278]
Please note this is to ensure the idea is captured, not to expedite implementation. Thanks.

**Idea captured during recent joint NSG-SC meeting:**

The Suggest a dataset form should try to capture the dataholder's country. 

This will help us flag the dataset with a "data-from-COUNTRY" flag. 

The geographic coverage field allows us to flag the dataset with a "data-about-COUNTRY" flag already. 

@siro1 @dschigel

**Comment by MortenHofft** [comment]
@kbraak Do I get this right: you would simply like to see a field below `Dataholders contact information` with `dataholder's country` similar to the `geographic coverage` field?

**Comment by kbraak** [comment]
That's correct. Ideally, these fields' values would be used to populate tags 'data-about-country' and 'data-from-country' - see https://github.com/gbif/data-mobilization/issues/61 as an example having both these tags. Thanks.
